models/intermediate/int_events__display_names.sql

```diff
@@ -0,0 +1,18 @@
+with events as (
+select
+    full_session_id,
+    session_id,
+    device_id,
+    view_id,
+    event_time,
+    updated_time,
+    processed_time,
+    user_display_name
+from {{ ref("stg_fullstory__events") }}
+where
+    user_display_name is not null
+    and full_session_id_rn = 1
```

Comment:
I'm pretty sure the only time a `display_name` is set is on the `identify` event, maybe an extra filter here?

Reply:
testing - disregard

Reply:
testing